rebar.config

```diff
@@ -44,7 +44,7 @@
         {idna, {git, "https://github.com/aeternity/erlang-idna", {ref, "24bf647"}}}
        ]}.
 
-{plugins, [{swagger_endpoints, {git, "https://github.com/aeternity/swagger_endpoints", {tag, "v0.1.0"}}}]}.
+{plugins, [{swagger_endpoints, {git, "https://github.com/aeternity/swagger_endpoints", {branch, "v0.2.0-wip"}}}]}.
```

Comment:
TODO update to tag after merging: https://github.com/ThomasArts/swagger_endpoints/pull/3